gettsim/taxes/zu_verst_eink/freibeträge.py

```diff
@@ -119,22 +121,22 @@ def eink_st_altersfreib(
     -------
 
     """
-    out = bruttolohn_m * 0
     agelimit = eink_st_abzüge_params["altersentlastungsbetrag_altersgrenze"]
```

Comment:
```suggestion
    altersgrenze = eink_st_abzüge_params["altersentlastungsbetrag_altersgrenze"]
```
No need to translate.